src/pages/home/report/ReportActionCompose.js

```diff
@@ -143,16 +143,17 @@ const {RNTextInputReset} = NativeModules;
  * Return the max available index for arrow manager.
  * @param {Number} numRows
  * @param {Boolean} isAutoSuggestionPickerLarge
+ * @param {Boolean} [isMentionSuggestion]
  * @returns {Number}
  */
-const getMaxArrowIndex = (numRows, isAutoSuggestionPickerLarge) => {
-    // EmojiRowCount is number of emoji suggestions. For small screen we can fit 3 items and for large we show up to 5 items
-    const emojiRowCount = isAutoSuggestionPickerLarge
-        ? Math.min(numRows, CONST.AUTO_COMPLETE_SUGGESTER.MAX_AMOUNT_OF_ITEMS)
-        : Math.min(numRows, CONST.AUTO_COMPLETE_SUGGESTER.MIN_AMOUNT_OF_ITEMS);
+const getMaxArrowIndex = (numRows, isAutoSuggestionPickerLarge, isMentionSuggestion = false) => {
```

Comment:
Maybe instead of `isMentionSuggestion` param, can we use `maxAmountOfItems = CONST.AUTO_COMPLETE_SUGGESTER.MAX_AMOUNT_OF_ITEMS` param and for mentions we would pass `CONST.AUTO_COMPLETE_SUGGESTER.MAX_AMOUNT_OF_MENTIONS`.

Seems to be cleaner. Or other suggestions also welcome, but the current change doesn't seem clean.

Reply:
@Nikhil-Vats

Reply:
@abdulrahuman5196 I agree, fixed.